src/ddsrt/include/dds/ddsrt/ifaddrs.h

```diff
@@ -36,15 +43,38 @@ struct ddsrt_ifaddrs {
 
 typedef struct ddsrt_ifaddrs ddsrt_ifaddrs_t;
 
+/**
+ * @brief Get the interface addresses (@ref ddsrt_ifaddrs) for specific address families
+ * 
+ * Note that array 'afs' must be terminated with an element 'DDSRT_AF_TERM'.
+ * 
+ * @param[out] ifap interface address pointer
+ * @param[in] afs an array of address families
```

Comment:
And this should state that the returned interfaces are restricted to the address families in `afs`.